jaxws-ri/rt/src/main/java/com/sun/xml/ws/util/xml/XMLStreamWriterFilter.java

```diff
@@ -72,7 +72,19 @@ private String transformWhiteSpaces(String text){
         return sb.toString();
     }
 
+    private char[] transformWhiteSpaces(char[] text, int start, int len){
+        for(int i=0; i<len; i++){
+                if (i+start >= text.length){
+                        break;
+                }
+                if(isUnusualChar(text[i]))
+                        text[i] = ' '; //change it to ' '
```

Comment:
As input `text` is modified here, why return it in line 83 and re-assign in line 87? I think the `void` return type would be better.